src/core/Header/Header.tsx

```diff
@@ -0,0 +1,163 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import '@itwin/itwinui-css/css/header.css';
```

Comment:
@mayank99 @veekeys I don't remember now but shouldn't this be like the last import?

Reply:
Yeah, was something...
I think we needed to put it after useTheme import hook...?

Reply:
Moved for all the files